### PR TITLE
Add more password messages

### PR DESCRIPTION
**Pull request by thanhz**
Your password must contain a minimum of 8 characters
Your password must include at least 1 uppercase character
Your password must include at least 1 number

Relies on the policies being created first https://github.com/nationalarchives/tdr-configurations/pull/123

### CODE REVIEW

File: themes/tdr/login/messages/messages_en.properties [hunk 1]
```diff
@@ -20,7 +20,10 @@ loginTotpUnableToScan=I want to enter a key instead
 mainHeader=Transfer Digital Records
 passwordNew=New password
 passwordReset=Password reset
-passwordRestrictions=Password must be at least 10 characters
+passwordRestrictions=Password must be at least 8 characters
+invalidPasswordMinLengthMessage=Invalid password: Your password must contain a minimum of {0} characters
+invalidPasswordMinUpperCaseCharsMessage=Invalid password: Your password must include at least {0} uppercase character
```
Comment (TomJKing) [inline themes/tdr/login/messages/messages_en.properties:25]: Should "character" be plural, ie "characters", or "character(s)"?

File: themes/tdr/login/messages/messages_en.properties [hunk 2]
```diff
@@ -20,7 +20,10 @@ loginTotpUnableToScan=I want to enter a key instead
...
+invalidPasswordMinDigitsMessage=Invalid password: Your password must include at least {0} number
```
Comment (TomJKing) [inline themes/tdr/login/messages/messages_en.properties:26]: Should "number" be plural, ie "numbers", or "number(s)"?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor questions

File: themes/tdr/login/messages/messages_en.properties (hunk 2 above)
Comment (thanhz) [reply themes/tdr/login/messages/messages_en.properties:26]: In the final design it is shown as just "number"

File: themes/tdr/login/messages/messages_en.properties (hunk 1 above)
Comment (thanhz) [reply themes/tdr/login/messages/messages_en.properties:25]: The final design has it as "uppercase character" and "minimum of 8 characters"

Comment (TomJKing) [reply themes/tdr/login/messages/messages_en.properties:25]: OK. Fair enough

File: themes/tdr/login/messages/messages_en.properties (hunk 2 above)
Comment (TomJKing) [reply themes/tdr/login/messages/messages_en.properties:26]: OK. Fair enough

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**